authorization-common/src/main/resources/actions.json

```diff
@@ -142,5 +142,41 @@
     "resourceType": "IMAGE_CATALOG",
     "actionType": "RESOURCE_DEPENDENT",
     "legacyRight": "datahub/write"
+  },
+  "CREATE_CLUSTER_DEFINITION": {
+    "right": "environments/createClusterDefinitions",
```

Comment:
Why not `"environments/createClusterDefinition"`? Other rights follow this convention.

Reply:
This name was specified in the jira issue, and the thunderhead rights were created according to that